apis/slo/v1alpha1/nodemetric_types.go

```diff
@@ -24,6 +24,15 @@ import (
 // EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
+// +kubebuilder:validation:Enum=usageWithHotPageCache;usageWithoutPageCache;usageWithPageCache
+type NodeMemoryCollectPolicy string
```

Comment:
We should add a comment here to define the default policy when the `nodeMetric.spec.nodeAggregatePolicy == nil`.